clients/client-python/build.gradle.kts

```diff
@@ -79,4 +85,11 @@ tasks {
       deleteCacheDir("__pycache__")
     }
   }
+
+  matching {
+    it.name.endsWith("envSetup")
+  }.all {
+    // add install package and code formatting before any tasks
+    finalizedBy(pipInstall, black, pylint)
+  }
```

Comment:
Which mean are these codes?

Reply:
hi @noidname01 I didn't found `envSetup` task in you PR.

Reply:
The `envSetup` task is from `VenvTask`, this task is respnsible for python virtual environment setup, so I think it's a great entry to bind our pipeline before running all the other tasks.
The source code is [here](https://github.com/PrzemyslawSwiderski/python-gradle-plugin/blob/7ee035aef2e25cee434364807518d132579d9f2e/src/main/kotlin/com/pswidersk/gradle/python/PythonPlugin.kt#L18-L21).